examples/tokenvm/storage/storage.go

```diff
@@ -40,10 +41,10 @@ type ReadState func(context.Context, [][]byte) ([][]byte, []error)
 // 0x5/ (hypersdk-fee)
 
 const (
-	// metaDB
+	// VM state
 	txPrefix = 0x0
 
-	// stateDB
+	// State keys
```

Comment:
This naming confuses me a bit. I think the difference here is indexes (currently called meta, which I'd say is also a slight misnomer) and StateDB, which is really just the active state of the chain.

I also don't think we should use state keys here as it overlaps with the state keys specified by actions.

Could we rename these as:
```
// Indexes

// Active state
```

(sorry, can't add a suggestion here)

Reply:
Yeah I don't even know what `meta` means. I guess it means `metadata` but it should probably be called `txDB`.